src/jupyter_contrib_nbextensions/nbextensions/pyVariableInspector/main.js

```diff
@@ -0,0 +1,397 @@
+define(["require", "jquery", "base/js/namespace", 'services/config',
+    'base/js/utils', 'notebook/js/codecell'
+], function(require, $, IPython, configmod, utils, codecell) {
+
+    var Notebook = require('notebook/js/notebook').Notebook 
+    "use strict";
+    //require('https://cdnjs.cloudflare.com/ajax/libs/jquery.tablesorter/2.25.7/js/jquery.tablesorter.min.js')
+
+    // ...........Parameters configuration......................
+    // define default values for config parameters if they were not present in general settings (notebook.json)
+    var cfg = {
+        'window_display': false,
+        'lenName': 16,
+        'lenType': 16,
+        'lenVar': 40,
+        'types_to_exclude': ['module', 'function', 'builtin_function_or_method', 'instance', '_Feature']  
+    }
+
+    //.....................global variables....
+
+    var liveNotebook = !(typeof IPython == "undefined")
+
+    var st = {}
+    st.config_loaded = false;
+    st.extension_initialized = false;
+
+    function read_config(cfg, callback) { // read after nb is loaded
+        // create config object to load parameters
+        var base_url = utils.get_body_data("baseUrl");
+        var initial_cfg = $.extend(true, {}, cfg);
+        var config = new configmod.ConfigSection('notebook', { base_url: base_url });
```

Comment:
instead of creating a new section, you actually `Jupyter.notebook.config` if you want the notebook config section.